CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The Product Changelog at **[matomo.org/changelog](https://matomo.org/changelog)*
 ### New APIs
 
 * Added new method `Piwik\API\Request::isRootRequestApiRequest()` to detect if the root request is an API request.
+* New event `Archive.noArchivedData` which can be used to execute code when no archive data is found for an API method.
```

Comment:
Can you remove the events from changelog again as we "ignore" them and don't consider them API?